CovidShield/mobile
# Add broadcast receiver to Android for e2e test on 2 devices

This PR adds broadcast receiver to Android Bridge, including: 
- Fix android bridge. 
- Add `TRANSMISSION_RISK_LEVEL` and hardcoded to 2 because this is the only use case of the app. 

Resolves https://github.com/CovidShield/mobile/issues/168

## How to test
- Run server locally and remove this condition https://github.com/CovidShield/server/blob/master/pkg/server/retrieve.go#L83 because the server doesn't return keys of current date.
- Install the app on 2 android devices. 
- 1st device set to positive by obtaining one time code. 
- Verify TEKs are uploaded successfully to server. 
- Open 2nd device and wait for exposed status. 
- Note that `ExposureNotificationClient.ACTION_EXPOSURE_STATE_UPDATED` broadcast receiver doesn't seem to be reliable. It's unpredictable to know when it is triggered. I received the callback a few times after almost 8 hours of waiting 🤔 

Below is one of the summary that I received after broadcast receiver. 
```
[Mon Jun 29 2020 18:32:46.872]  LOG      ccccc summary {"_summaryIdx": "7e72a837-8bfa-4630-ba85-7a77a4563913", "daysSinceLastExposure": 1, "matchedKeyCount": 3, "maximumRiskScore": 25}
```
